types/three/test/lights/lights-spotlight.ts

```diff
@@ -55,7 +55,7 @@ function init() {
 
     const geometry = new THREE.PlaneGeometry(2000, 2000);
 
-    let mesh = new THREE.Mesh(geometry, material);
+    const mesh = new THREE.Mesh(geometry, material);
```

Comment:
Can you add a new test that tests this change more specifically (i.e., something that would require a cast without generics, but doesn't require a cast with generics)?